### PR TITLE
Document @cloudflare/pages-plugin-vercel-og

### PR DESCRIPTION
**Pull request by GregBrimble**
https://432bbf77.cloudflare-docs-7ou.pages.dev/pages/platform/functions/plugins/vercel-og/

### CODE REVIEW

File: content/pages/platform/functions/plugins/vercel-og.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+pcx_content_type: concept
```
Comment (WalshyDev) [inline content/pages/platform/functions/plugins/vercel-og.md]:
```suggestion
pcx_content_type: reference
```

File: content/pages/platform/functions/plugins/vercel-og.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+title: vercel/og
+weight: 1
```
Comment (WalshyDev) [inline content/pages/platform/functions/plugins/vercel-og.md]:
```suggestion
```

File: content/pages/platform/functions/plugins/vercel-og.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+import React from "react";
+import vercelOGPagesPlugin from "@cloudflare/pages-plugin-vercel-og";
+
+interface Props {
+	ogTitle: string;
+}
+
+export const onRequest = vercelOGPagesPlugin<Props>({
+	imagePathSuffix: "/social-image.png",
+	component: ({ ogTitle, pathname }) => {
```
Comment (WalshyDev) [inline content/pages/platform/functions/plugins/vercel-og.md:33]: nit: pathname doesn't look to be used
```suggestion
	component: ({ ogTitle }) => {
```

File: content/pages/platform/functions/plugins/vercel-og.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+- `onError`: an optional function which returns a `Response` or a promise of a `Response`. This function is called when a request is made to the `imagePathSuffix` and `extractors` are provided but the underlying webpage is not valid HTML. Defaults to returning a 404 response.
+
+- `autoInject`: an optional object with an optional property: `openGraph`. If set to `true`, the Plugin will automatically set the `og:image`, `og:image:height` and `og:image:width` meta tags on the underlying webpage.
+
+### Generating arbitrary images
+
+You can use this Plugin's API to generate arbitrary images, not just as middleware.
+
+For example:
```
Comment (WalshyDev) [inline content/pages/platform/functions/plugins/vercel-og.md]:
```suggestion
For example, the below code will generate an image saying "Hello, world!" which is available at `/greet`.
```

**Review by WalshyDev — APPROVED** [review]
Nice! This is super cool

File: content/pages/platform/functions/plugins/vercel-og.md (hunk 3 above)
Comment (GregBrimble) [reply content/pages/platform/functions/plugins/vercel-og.md:33]: Just demonstrating that it's there and available to use :)

**Pull request merged**